docs/semgrep-ci/configuring-blocking-and-errors-in-ci.md

```diff
@@ -26,7 +26,7 @@ When Semgrep identifies one or more blocking findings, it returns exit code `1`.
 
 The process to enforce a block on a PR or MR after Semgrep exits with error code `1` is dependent on your CI provider. Review your CI provider's documentation for further information.
 
-If Semgrep encounters an internal error, it sends an anonymous crash report to a crash-reporting server and returns exit code `2`. Review your CI provider's documentation for further information on how to handle this error code, such as failing the pipeline job. 
+If Semgrep encounters an internal error, it sends an anonymous crash report to a crash-reporting server. By default, it returns exit code 0 in CI. If you would prefer to catch internal errors, review the options below and the Semgrep exit codes reference to determine how you want to handle each exit code.
```

Comment:
```suggestion
If Semgrep encounters an internal error, it sends an anonymous crash report to a crash-reporting server. By default, it returns exit code 0 in CI. If you would prefer to catch internal errors, review the options below and the Semgrep exit codes reference to determine how you want to handle each exit code.
```

Maybe something like "If Semgrep encounters an internal error, it sends an anonymous crash report to a crash-reporting server and returns exit code `0`. If you want to catch internal errors, review the [CLI reference](/cli-reference#exit-codes) for more information about Semgrep's exit codes and the options explained in this article to determine how you want to handle each exit code." and then remove the admonition that immediately follows since you're linking to the CLI docs in the text itself

Reply:
That makes sense to me - are you open to making the changes? Or should I just commit all the suggestions to get that result? (Sorry, brain is a little fried this week.)

Reply:
No worries -- my brain is mushy this week, too. I'm sure there's a better way to suggest multi-line changes, but I can't figure it out today. I'll push a commit with my suggestions